### PR TITLE
chore(e2e): fix e2e tests

### PR DESCRIPTION
**Pull request by PatAKnight**
## Description

There were recent updates to the showcase repo has lead to many failing e2e tests. The changes included aim to fix the problems that we are facing.

## Which issue(s) does this PR fix

- Fixes [RHIDP-4775](https://issues.redhat.com/browse/RHIDP-4775)

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Review by gashcrumb — APPROVED** [review]

**Review by gashcrumb — APPROVED** [review]

**Review by nickboldt — APPROVED** [review]
looks reasonable to me

File: dynamic-plugins/wrappers/immobiliarelabs-backstage-plugin-gitlab-backend-dynamic/src/dynamic/index.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import type { BackendDynamicPluginInstaller } from '@backstage/backend-dynamic-feature-service';
+import { createBackendFeatureLoader } from "@backstage/backend-plugin-api";
```
Comment (schultzp2020) [inline dynamic-plugins/wrappers/immobiliarelabs-backstage-plugin-gitlab-backend-dynamic/src/dynamic/index.ts:1]: I don't see any related wrapper package.json changes.

**Review by schultzp2020 — CHANGES_REQUESTED** [review]

**Comment by nilgaar** [comment]
I see that we reached test execution, so that's and improvement (https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/janus-idp_backstage-showcase/1869/pull-ci-janus-idp-backstage-showcase-main-e2e-tests/1853429130473771008/artifacts/e2e-tests/janus-idp-backstage-showcase/build-log.txt).

But RHDH can't load the login view: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/janus-idp_backstage-showcase/1869/pull-ci-janus-idp-backstage-showcase-main-e2e-tests/1853429130473771008/artifacts/e2e-tests/janus-idp-backstage-showcase/artifacts/showcase/index.html @PatAKnight any idea?

File: .ibm/pipelines/value_files/values_showcase.yaml [hunk 2]
```diff
@@ -52,10 +52,10 @@ global:
           kubernetes:
             clusterLocatorMethods:
               - clusters:
-                - authProvider: serviceAccount
-                  name: 'my-cluster'
-                  serviceAccountToken: ${K8S_CLUSTER_TOKEN_ENCODED}
-                  url: ${K8S_CLUSTER_API_SERVER_URL}
+                  - authProvider: serviceAccount
```
Comment (gashcrumb) [inline .ibm/pipelines/value_files/values_showcase.yaml:55]: Were these extra spaces actually needed?  There's a few changes in these files which seem more like a different tab width setting.  Not necessarily a big deal (except YAML) but does create extra unnecessary changes.

**Review by josephca — COMMENTED** [review]
/lgtm

**Pull request merged**